new/package.json

```diff
@@ -15,7 +15,7 @@
     "prettier": "3.3.0",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
-    "react-scripts": "5.0.1",
+    "react-scripts": "^5.0.1",
```

Comment:
Any reason in particular for this change?

Reply:
Hmm this one was automatically added when I tried to install react-scripts.